sdk/test/Services/DynamoDBv2/IntegrationTests/DataModelTests.cs

```diff
@@ -145,7 +145,9 @@ public void TestContext_DisableFetchingTableMetadata_DateTimeAsHashKey()
                 NonEpochDate1 = EpochDate,
                 NonEpochDate2 = EpochDate,
                 LongEpochDate1 = LongEpochDate,
-                LongEpochDate2 = LongEpochDate.AddDays(12)
+                LongEpochDate2 = LongEpochDate.AddDays(12),
+                NullableLongEpochDate1 = null,
```

Comment:
Should the non long EpochDate be checked for nullability also?  I see differences in the code of the non long vs the long date where I would think the code would be similar. This same comment applies to all the tests below.

Reply:
Ideally we should add the check for non-long epoch date type. The missing test was pointed in the PR. It currently works since in case of exception we return the original entry.

Please advise if we should increase the scope of PR by adding test check for old deprecated `StoreAsEpoch` attribute.